list/List.js

```diff
@@ -227,9 +225,14 @@ define([
 		 * @private
 		 */
 
-		render: function () {
+		postRender: function () {
```

Comment:
I think you are right, because delite/Widget doesn't call `initializeInvalidating()` until `postRender()`.